dataset-snippets/join_dataframes.py

```diff
@@ -0,0 +1,11 @@
+# Use snippet 'join_dataframes' to join together two dataframes.
+# It assumes the 'Setup' snippet has been executed.
+#
+# In the example below, it joins Demographics '_person_df' and Measurements '_measurement_df' using
+# any columns they have in common, which in this case should only be 'person_id'.
+#
+# See also https://pandas.pydata.org/pandas-docs/version/0.25.1/reference/api/pandas.merge.html
+
+
+## -----[ CHANGE THE DATAFRAME NAME(S) TO MATCH YOURS FROM DATASET BUILDER] -----
+measurement_df = pd.merge(left=YOUR_DATASET_NAME_person_df, right=YOUR_DATASET_NAME_measurement_df, how='inner')
```

Comment:
FYI, generated dataset variable names should be changing soon to be less interesting. There were a number of unpleasant edge cases when trying to map a free text name to a python or R variable directly, so it's going to be something formulaic like`dataset_{random_number}_{domain_name}_df`. Would it be useful to provide a frame name(s) directly in the dataset builder that can be exported to the notebook?

If you have any feature requests for the Dataset builder, especially around making it more natural to use with realistic notebooks, we'd be interested to hear those.

Reply:
Thanks for the heads up! Sent some suggestions offline.